packages/teams-js/src/public/pages.ts

```diff
@@ -750,4 +750,94 @@ export namespace pages {
       return runtime.supports.pages ? (runtime.supports.pages.appButton ? true : false) : false;
     }
   }
+
+  /**
+   * Parameters for the NavigatewithinApp API
+   *
+   * @beta
+   */
+  export interface NavigateWithinAppParams {
+    /**
+     * @deprecated
+     * As of 2.0.0, please use {@link app.PageInfo.id | app.Context.page.id} instead
+     *
+     * Developer-defined ID of the Page to navigate to within the application (Formerly EntityID)
+     */
+    pageId: string;
+
+    /**
+     * @deprecated
+     * As of 2.0.0, please use {@link app.PageInfo.subPageId | app.Context.page.subPageId} instead
+     *
+     * Optional developer-defined ID describing the content to navigate to within the Page. This will be passed
+     * back to the application via the Context object.
+     */
+    subPageId?: string;
+  }
+
+  /**
+   * Provides functions for navigating without your own application ID.
+   *
+   * @beta
+   */
+  export namespace navigate {
+    /**
+     * Navigate to the currently running application with page ID, and sub-page ID (for navigating to
+     * specific content within the page). This is equivalent to navigating to a deep link with the above data, but
```

Comment:
I think we can remove everything from "This is equivalent to..." to "...formats for different hosts". I think we want to stop encouraging people to use (or have to think about) deep links, and I don't think the extra detail here helps users understand how to use the function. I would defer to Saurabh and Erika's opinions though

Reply:
I agree about taking a forward-facing approach to our reference docs. Compat explanations we can cover in the deeplink conceptual docs--we have a workitem to refactor them with latest best practice.

Reply:
Resolved in latest commit